packages/electron-commons/src/initializers/electron-node.ts

```diff
@@ -12,14 +12,29 @@ export interface InitializeNodeEnvironmentOptions extends InitializerOptions {
     processOptions?: Pick<SpawnOptions, 'cwd' | 'shell' | 'env' | 'stdio' | 'windowsHide'>;
 }
 
+export interface DisconnectHandler {
+    (details: DisconnectDetails): void;
+}
+
+export interface DisconnectDetails {
```

Comment:
I'd rather call it `ProcessExitDetails` because 1) it's about process, and 2) it's about exiting.

`disconnect` and `exit` both exist, but they're different events:
* https://nodejs.org/api/process.html#event-disconnect
* https://nodejs.org/api/process.html#event-exit

I would also remove `DisconnectHandler`, and instead use `(details: ProcessExitDetails) => void` inline to make it more self-descriptive, it wouldn't be too much boilerplate.

Reply:
thanks, renamed